code/modules/power/supermatter/supermatter.dm

```diff
@@ -567,10 +567,12 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 			l.hallucination += power * config_hallucination_power * D
 			l.hallucination = clamp(l.hallucination, 0, 200)
 
-	for(var/mob/living/l in range(src, round((power / 100) ** 0.25)))
+	/*for(var/mob/living/l in range(src, round((power / 100) ** 0.25)))
```

Comment:
?

Reply:
It literally bypasses the glass insulation thing, and the code isnt needed as there is radiation_pulse(max_rads) in sm.dm

Reply:
Any mob inside the range of 10 tiles from even behind 5 reinforced wall still can be fried to dead

Reply:
well then remove the code?

Reply:
yeah